media-gfx/flameshot/flameshot-0.10.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop xdg
+
+DESCRIPTION="Powerful yet simple to use screenshot software"
+HOMEPAGE="https://flameshot.org https://github.com/flameshot-org/flameshot"
+SRC_URI="https://github.com/flameshot-org/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0 Free-Art-1.3 GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	=dev-qt/qtsingleapplication-2.6*[qt5(+),X]
+	dev-qt/qtwidgets:5
+	dev-qt/qtsvg:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtdbus:5
+	sys-apps/dbus
+	dev-libs/spdlog
+"
+BDEPEND="
+	dev-qt/linguist-tools:5
+"
+RDEPEND="${DEPEND}"
+PATCHES=(
+	"${FILESDIR}/${PN}-0.9.0-dont-force-ccache.patch"
+)
+
+src_prepare() {
```

Comment:
Please remove external packages that we unbundle just to make sure our unbundling really works

Reply:
See flameshot-0.9.0 ebuild

Reply:
i tested it to be sure it works and saw no need for it to be there since the path for version `0.9.0`'s `CMakeLists.txt` was implemented in this release.